doc/source/What Programming Language to choose as a beginner.md

```diff
@@ -0,0 +1,28 @@
+## What Programming Language to choose as a beginner?  
```

Comment:
Main title should start with only one hashtag (#)

Reply:
Also you will need an emoji for the title

Reply:
Thanks for reviewing,
I have done the changes. Please check again.